content/env-variables.md

```diff
@@ -11,16 +11,16 @@ objectives:
 
 -   There are no "out of the box" solutions for creating distinct environments in an onchain program, but you can achieve something similar to environment variables if you get creative.
 -   You can use the `cfg` attribute with **Rust features** (`#[cfg(feature = ...)]`) to run different code or provide different variable values based on the Rust feature provided. _This happens at compile-time and doesn't allow you to swap values after a program has been deployed_.
--   Similarly, you can use the `cfg!` **macro** to compile different code paths based on the features that are enabled.
--   Alternatively, you can achieve something similar to environment variables that can be modified after deployment by creating accounts and instructions that are only accessible by the program’s upgrade authority.
+-   Similarly, you can use the `cfg!` **macro** to compile different code paths based on the enabled features.
+-   Alternatively, you can achieve something similar to environment variables that can be modified after deployment by creating accounts and instructions only accessible by the program’s upgrade authority.
 
 # Lesson
 
-One of the difficulties engineers face across all types of software development is that of writing testable code and creating distinct environments for local development, testing, production, etc.
+One of the difficulties engineers face across all types of software development is writing testable code and creating distinct environments for local development, testing, production, etc.
 
 This can be particularly difficult in Solana program development. For example, imagine creating an NFT staking program that rewards each staked NFT with 10 reward tokens per day. How do you test the ability to claim rewards when tests run in a few hundred milliseconds, not nearly long enough to earn rewards?
 
-Traditional web development solves some of this with environment variables whose values can differ in each distinct "environment." Currently, there's no formal concept of environment variables in a Solana program. If there were, you could just make it so that rewards in your test environment are 10,000,000 tokens per day and it would be easier to test the ability to claim rewards.
+Traditional web development solves some of this with environment variables whose values can differ in each distinct "environment." Currently, there's no formal concept of environment variables in a Solana program. If there were, you could just make it so that rewards in your test environment are 10,000,000 tokens per day, and it would be easier to test the ability to claim rewards.
```

Comment:
Let's make this added comma a period so it'll be:
> If there were, you could make it so that rewards in your test environment are 10,000,000 tokens per day. Then it would be easier to test the ability to claim rewards.